views/partials/navbar.ejs

```diff
@@ -2,7 +2,7 @@
   <a href="/">
     <img
       class="shine-logo"
-      src="https://images.ctfassets.net/awpxl2koull4/FFhnW3pbga4qmCOM24kmc/8c718dac9a5357db86c55b637461f4f3/Shine-logo-web.png?w=100"
+      src="https://images.ctfassets.net/awpxl2koull4/4VYfJ5qvYsWIM0e8k2SWAM/a02d1d7db9166a3c20b42352480f506d/Shine-Logo-no-circle.png?w=100"
```

Comment:
Maybe try out this one:
```suggestion
       src="https://images.ctfassets.net/awpxl2koull4/6Ge2cCqfKg0IM8CIsOMiq6/bc0dee5dd91c1b37a155c220e95893d4/shine-logo-nav-122018.png?w=100"
```

It should look like...
![screen shot 2018-12-22 at 12 17 12 am](https://user-images.githubusercontent.com/696595/50370950-04bd2180-057f-11e9-93cb-6a2c067e7073.png)